<!-- -*- mode: XML -*- -->
<launch>
  <arg name="time"/>
  <node pkg="gummi_ankle" name="imu_data_acquisition" type="imu_data_acquisition.py" output ='screen'/>
  <node pkg="gummi_ankle" name="export_imu_fsr_data" type="export_imu_fsr_data.py" output ='screen'/>
  <node pkg="gummi_ankle" name="threshold_gait_detection_node" type="threshold_gait_detection_node" output ='screen'/>
  <node pkg="gummi_ankle" name="gait_assistance" type="gait_assistance.py" output='screen' args="-t $(arg time)"/>
</launch>
